torch/_torch_docs.py

```diff
@@ -4050,6 +4050,35 @@ def merge_dicts(*dicts):
             [5, 6, 7, 8]])
 """.format(**common_args))
 
+add_docstr(torch.unflatten,
+           r"""
+unflatten(input, dim, sizes) -> Tensor
+
+Expands a dimension of the input tensor over multiple dimensions.
+
+A tensor reshaping function that does the opposite of :func:`torch.flatten`.
```

Comment:
What about using `seealso`?
```suggestion
.. seealso::

    :func:`torch.flatten` the inverse of this function. It coalesces several dimensions into one.
```